### ISSUE TITLE
Track errors in the frontend and backend

### ISSUE THREAD

**Issue by BenJeau** [CDH-Studio/I-Talent#256]
**Is your feature request related to a problem? Please describe.**
If errors are happening in the frontend or the backend, there's no central place to track them, and our developers do not know if they are happening for the client.

**Describe the solution you'd like**
A central place keeping a log of issues/errors.

**Describe alternatives you've considered**
There's these tools available
- Grafana/Prometheus https://medium.com/teamzerolabs/node-js-monitoring-with-prometheus-grafana-3056362ccb80 (open source)
- Bugsnag (paid)
- sentry.io (open source and paid if hosted on their servers)

**Comment by BenJeau** [comment]
Status: waiting on getting access to openshift to deploy a sentry.io server

**Comment by BenJeau** [comment]
Put on hold since sentry.io doesn't really work in a Kubernetes environment. Looking for alternatives while focusing on bugs right now.

**Comment by BenJeau** [comment]
Here's more information on Prometheus and how to use it in node.js https://github.com/RisingStack/example-prometheus-nodejs

**Comment by mo-5** [comment]
For this, we can do something similar to this https://github.com/CDH-Studio/I-Talent/pull/1059 when an error is faced on the frontend to push the error to the database and we will be able to look at them afterwards and see if they require attention.

**Comment by BenJeau** [comment]
Honestly that is probably the simplest solution, if we do that, we should invest more time in making that error catching in the frontend good, since I don't think its bullet proof. (We should implement Error Bondaries for React components)

**Comment by BenJeau** [comment]
@MohamedRadwan feel free to remove me from this issue and assign it to yourself if you want, I'll focus on IE for this week

**Comment by BenJeau** [comment]
But I think we should maybe have a Prometheus instance? Not 100% sure

**Comment by mo-5** [comment]
Duplicate of issue #344
